### PR TITLE
Update O365 readme file 

### PR DESCRIPTION
**Pull request by GinnyGalicia**
Type of change

- Enhancement

## What does this PR do?

Tidy up of Markdown in. Readme.md, including adding link todeveloper's documentation, if missing 

## Checklist

- [ ] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [ ] I have verified that all data streams collect metrics or logs.
- [x] I have added an entry to my package's `changelog.yml` file.
- [ ] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

## Author's Checklist

- [ ]

## Related issues

-

### CODE REVIEW

File: packages/1password/_dev/build/docs/README.md [hunk 1]
```diff
@@ -34,4 +34,4 @@ Uses the 1Password Events API to retrieve information about items in shared vaul
 
 {{fields "item_usages"}}
 
-{{event "item_usages"}}
+{{event "item_usages"}}
```
Comment (efd6) [inline packages/1password/_dev/build/docs/README.md:37]: Please revert this.

Comment (GinnyGalicia) [reply packages/1password/_dev/build/docs/README.md]: Done

**Review by r00tu53r — COMMENTED** [review]
The field level doc changes shouldn't be part of this PR as there were no changes related to those. Please refer to my comment [here](https://github.com/elastic/integrations/pull/3063#pullrequestreview-940452252) and [here](https://github.com/elastic/integrations/pull/3063#issuecomment-1097882458). Please apply those steps and rebuild.

**Pull request closed without merging**